disk_usage.rst

```diff
@@ -2,90 +2,129 @@
+   unit. Verify that the device remains mounted after a reboot.
+
+3. Configure the node agent to use `/home1` as the base directory for new
+   applications: ::
+
+    runagent -m node configure-home-basedir --set /home1
+
+Migrate /home data to a new disk
+--------------------------------
+
+If the previous section is not applicable, consider adding a new disk and
+migrating existing data with a Bash script included in the core:
+``migrate-home-disk.sh``. This script illustrates the basic steps required
+for data migration.
+
+.. warning::
+
+  The included script is just an example that may not fit your system. USE
+  AT YOUR OWN RISK!
 
-If you can't do any of the above, consider adding a new disk and migrating existing data using the script below.
 The script will:
 
-* stop all rootless containers
-* copy all rootless containers inside the new disk
+* stop all rootless applications
+* copy all rootless applications inside the new disk
 * reclaim space from the root filesystem
 * mount the new disk under ``/home``
-* restart all rootless containers
-
-Before running the script, make sure to attach the disk to the machine, format it and mount to a custom
-location like ``/mnt/temp_disk``.
-Then launch the script by passing the mount location as parameter, like: ::
-
-  migrate /mnt/temp_disk
-
-The ``migrate`` script:.
-
-.. code-block:: bash
-
-  #!/bin/bash
-  set -e
-  mount_dir=$1
-  if [ -z "$mount_dir" ]; then
-    echo "Please provide the disk mount dir"
-    exit 1
-  fi
-  if [ ! -d "$mount_dir" ]; then
-    echo "Mount dir not found"
-    exit 1
-  fi
-  if ! which rsync &>/dev/null; then
-    echo "rsync not found, please install it before proceed"
-    exit 1
-  fi
-  device=$(grep $(echo "$mount_dir" | sed 's/\/$//') /etc/mtab | awk '{print $1}')
-  if [ -z "$device" ]; then
-    echo "Device not found for $mount_dir"
-    exit 1
-  fi
-  for userhome in /home/*[0-9]; do
-    moduleid=$(basename $userhome)
-    echo "Disabling rootless module ${moduleid}..."
-    systemctl stop user@$(id -u $moduleid)
-  done
-  echo "Copying files..."
-  rsync -avrX --delete-after /home/ $mount_dir/
-  echo "Reclaim space..."
-  for userhome in /home/*[0-9]; do
-   rm -rf $userhome
-  done
-  echo "Remount disk to /home..."
-  umount $device
-  mount $device /home
-  for userhome in /home/*[0-9]; do
-    moduleid=$(basename $userhome)
-    echo "Starting rootless module ${moduleid}..."
-    systemctl start user@$(id -u $moduleid)
-  done
-  echo "Done"
-  echo
-  eval $(blkid $device -o export)
-  echo "Add this line to fstab:"
-  echo
-  echo "UUID=$UUID /home $TYPE rw,errors=remount-ro 0 1"
+* restart all rootless applications
+
+Before running the script, make sure to attach the disk to the node,
+format it, and mount it to a custom location like ``/mnt/temp_disk``.
+
+Then launch the Bash script by passing the mount location as a parameter,
+like: ::
 
+  bash /var/lib/nethserver/node/migrate-home-disk.sh /mnt/temp_disk
```

Comment:
```suggestion
  migrate-home-disk.sh /mnt/temp_disk
  
Follow the on-screen instructions provided by the script.
```

Reply:
@nrauso what do you think?

Reply:
I prefer @DavidePrincipi's approach: it’s a highly advanced customization, so I’d handle it the same way we do with the uninstall script.

Reply:
:ok: